### ISSUE TITLE
[Beta] Span attributes are not showing in application insights

### ISSUE THREAD

**Issue by marcus13371337** [microsoft/ApplicationInsights-node.js#1108]
Hello!

I'm using the latest version of the beta-version and I can't see the custom attributes on my spans. I read the documentation [here](https://github.com/microsoft/ApplicationInsights-node.js/tree/beta#add-span-attributes), and to my understanding everything in "attributes" of a span should be visible in the "customAttributes"-field in the dependency graph in application-insights.

As example, if I change the span-processor to log out to the console (with the following code):
```ts
  const config = new ApplicationInsightsConfig();

  config.connectionString = ENV.APPLICATIONINSIGHTS_CONNECTION_STRING;
  config.instrumentations = {
    http: {
      enabled: false,
    },
    postgreSql: {
      enabled: false,
    },
  };
  config.samplingRatio = 1;

  appInsights = new ApplicationInsightsClient(config);

  const traceHandler = appInsights.getTraceHandler();

  traceHandler.addInstrumentation(new PrismaInstrumentation());

      traceHandler.addSpanProcessor(
        new SimpleSpanProcessor(new ConsoleSpanExporter())
      );

  appInsights.start();

```

I can see the following spans in the console:
```
{
  traceId: 'f636013c939fd461c0a3ab4a4f0b9638',
  parentId: '83c700ee8ba2182a',
  traceState: undefined,
  name: 'prisma:engine:db_query',
  id: '8a514e4603a18de0',
  kind: 0,
  timestamp: 1679401447754670,
  duration: 17380,
  attributes: {
    'db.statement': 'SELECT COUNT(*) FROM XXXXX'
  },
  status: { code: 0 },
  events: [],
  links: []
}
```

However, if I check that dependency in application insights, there are no attributes attached to the span, what am I doing wrong?

**Comment by marcus13371337** [comment]
I think I found the issue, it seems as if the `AzureSpanExporter` requires that the span have the `SemanticAttributes.DB_SYSTEM` in order to pick up any `db.*`-attributes

https://github.com/Azure/azure-sdk-for-js/blob/2d6f28bf1e8fe390412b1859060151710f02b1c7/sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts#L170

That's not really how it's described in your documentation...

**Comment by marcus13371337** [comment]
Issue reported here!
https://github.com/Azure/azure-sdk-for-js/issues/25322#issue-1633880020

**Comment by hectorhdzg** [comment]
@marcus13371337 current behavior is by design and was based on current OpenTelemetry specification, we use specific span attributes to build other properties of the telemetry event depending on the type, ex: db.stament should be part of Dependency data field in Application Insights, like you mentioned db.system should be present for Azure Monitor Exporter to consider that span a DB dependency telemetry event

**Comment by marcus13371337** [comment]
@hectorhdzg thanks for your reply and your knowledge sharing. The reason for why I thought that all attributes should be visible in applicationinsights was because of this paet of your documentation:
``md
Add a custom property to a Trace

Any [attributes](https://github.com/microsoft/ApplicationInsights-node.js/issues/1108#add-span-attributes) you add to spans are exported as custom properties. They populate the customDimensions field in the requests or the dependencies tables in Application Insights.
``

that seems a bit off from what you described/my findings, or am I missinterpeting the documentation?

**Comment by hectorhdzg** [comment]
@marcus13371337 yeah documentation could be improved here, you can in fact add a custom property but it cannot have existing semantic conventions names, like "db.*"  we will have a sync next week on our side to discuss this topic, I will update this issue after that.
